src/sentry/utils/performance_issues/detectors/uncompressed_asset_detector.py

```diff
@@ -76,7 +79,9 @@ def visit_span(self, span: Span) -> None:
             return
 
         # Ignore assets with certain file extensions
-        if not description.endswith(FILE_EXTENSION_ALLOWLIST):
+        normalized_description = description.strip().upper()
+        extension = EXTENSION_REGEX.search(normalized_description)
+        if extension and extension.group(1) not in FILE_EXTENSION_ALLOWLIST:
```

Comment:
This is the same regex and pattern used in the Large HTTP Payload detector here:
https://github.com/getsentry/sentry/pull/49703/files